domains/sbdesigns.json

```
@@ -4,7 +4,13 @@
     "email": "sebandblocks@turquoisetnt.one"
   },
   "records": {
-    "CNAME": "sbdesigns.pages.dev"
+    "CNAME": "sbdesigns.pages.dev",
+    "MX": [
+      { "target": "mx.zoho.eu", "priority": 10 },
+      { "target": "mx2.example.com", "priority": 20 },
+      { "target": "mx3.example.com", "priority": 50 }
+    ],
```

```suggestion
    "MX": ["mx.zoho.eu"],
```
mx*.example.com -> invalid mx records
mx.zoho.eu -> the default priority is 10